libs/react-helpers/src/hooks/use-data-provider.ts

```diff
@@ -85,7 +86,7 @@ export const useDataProvider = <
         isInsert,
         isUpdate,
       } = args;
-      setError(error);
+      setError(isNotFoundGraphQLError(error) ? undefined : error);
```

Comment:
chore (blocking): I'd revert this, the data provider should still return any error it sees, it should be up to the application/component to decide how to handle it

Reply:
agree!!! updated in last commit.